tests/intergration/test_stream.py

```diff
@@ -0,0 +1,89 @@
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import pytest
+from gqlalchemy import MemgraphKafkaStream, MemgraphPulsarStream
+
+
+def stream_exists(streams, stream_name):
+    return any(map(lambda s: s["name"] == stream_name, streams))
+
+
+def test_create_kafka_stream(memgraph):
+    kafka_stream = MemgraphKafkaStream(name="test_stream", topics=["topic"], transform="kafka_stream.transform")
+
+    with pytest.raises(Exception) as e_info:
+        memgraph.create_stream(kafka_stream)
+    assert "Local: Broker transport failure" in str(e_info.value)
+
+
+def test_drop_kafka_stream(memgraph):
+    kafka_stream = MemgraphKafkaStream(name="test_stream", topics=["topic"], transform="kafka_stream.transform")
+
+    with pytest.raises(Exception) as e_info:
+        memgraph.create_stream(kafka_stream)
+    assert "Local: Broker transport failure" in str(e_info.value)
+
+
+def test_create_pulsar_stream(memgraph):
+    pulsar_stream = MemgraphPulsarStream(name="test_stream", topics=["topic"], transform="pulsar_stream.transform")
+
+    with pytest.raises(Exception) as e_info:
+        memgraph.create_stream(pulsar_stream)
+    assert "Pulsar consumer test_stream : ConnectError" in str(e_info.value)
+
+
+def test_drop_pulsar_stream(memgraph):
+    pulsar_stream = MemgraphPulsarStream(name="test_stream", topics=["topic"], transform="pulsar_stream.transform")
+
+    with pytest.raises(Exception) as e_info:
+        memgraph.create_stream(pulsar_stream)
+    assert "Pulsar consumer test_stream : ConnectError" in str(e_info.value)
+
+
+def test_kafka_stream_cypher():
+    kafka_stream = MemgraphKafkaStream(name="test_stream", topics=["topic"], transform="kafka_stream.transform")
+    query = "CREATE KAFKA STREAM test_stream TOPICS topic TRANSFORM kafka_stream.transform;"
+    assert kafka_stream.to_cypher() == query
+
+
+def test_pulsar_stream_cypher():
+    pulsar_stream = MemgraphPulsarStream(name="test_stream", topics=["topic"], transform="pulsar_stream.transform")
+    query = "CREATE PULSAR STREAM test_stream TOPICS topic TRANSFORM pulsar_stream.transform;"
+    assert pulsar_stream.to_cypher() == query
+
+
+def test_kafka_stream_extended_cypher():
+    kafka_stream = MemgraphKafkaStream(
+        name="test_stream",
+        topics=["topic"],
+        transform="kafka_stream.transform",
+        consumer_group="my_group",
+        batch_interval="9999",
+        bootstrap_servers="'localhost:9092'",
+    )
+    query = "CREATE KAFKA STREAM test_stream TOPICS topic TRANSFORM kafka_stream.transform CONSUMER_GROUP my_group BATCH_INTERVAL 9999 BOOTSTRAP_SERVERS 'localhost:9092';"
+    assert kafka_stream.to_cypher() == query
```

Comment:
@g-despot try also to execute this query because that will cover most (actually the whole implementation) of `create_stream`, `get_streams`, and `drop_stream`

Reply:
Aha, that's tested before 😄

Reply:
The `get_streams` and `drop_stream` are functions in `Memgraph` that don't have a state, so you can't easily inspect the cypher they create. And it is not a reliable test if the implementation ever changes. These stream tests do not actually create any streams in Memgraph because mgclient throws:

```
mgclient.DatabaseError: Failed to initialize Kafka consumer test_stream : Local: Broker transport failure
```

I can add the test declarations so that future implementations don't forget to cover these cases.